release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -40,6 +40,43 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-7-complex-root-devices"]
+==== Complex root tooling for LUKS and RAID devices
+
+Disk tooling enhancements now support complex root devices in bare metal deployments. For example, software RAID improvements now allow for more robust nodes that can still reboot in the event of a drive failure. And support for multiple LUKS root devices means that you now have added capabilities such as using the Local Storage Operator in file mode, having your non-rootfs data encrypted at rest, and use of third-party software that requires local persistent storage.
```

Comment:
```suggestion
{op-system} 4.7 includes several improvements to disk provisioning for bare metal deployments.  Native Ignition support for LUKS disk encryption provides additional configurability for encrypted root filesystems, as well as support for encryption of additional data filesystems.  In addition, {op-system} now supports boot disk mirroring (except on s390x), providing redundancy in the case of disk failure.
```

Reply:
Nice, thanks for this! I'll use your suggested change. I've also dropped the bulleted list that follows.

Reply:
It looks as though this change didn't make it in?

Reply:
Sorry for the confusion, the change made it but I changed it to a list format (lines [48-57](https://github.com/openshift/openshift-docs/pull/28368/files#diff-14b6355bc5db567351b9e43a469c73396794e644c2dd1fc8523417f2f7b72c16R48-R57))

Reply:
Will remove the first paragraph